runbooks/deployCyphernodeAWS.md

```diff
@@ -0,0 +1,17 @@
+Run book how to create an online cyphernode instance
+
+1. Generate a key pair in ec2 console and download it https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#KeyPairs:
+1. connect to cloud formation https://console.aws.amazon.com/cloudformation/home?region=us-east-1
```

Comment:
Can you add more detailed steps on how to do this for those of us who never used couldformation?

Looking at the page at #2 the next step could look like : 
```
* In the `Prerequisite`, select `Template is ready`
* In the `Specify template`, select `Upload a template file`
```

Reply:
updated!